python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -568,10 +533,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "p = figure(x_axis_label='Mean DP', y_axis_label='Call Rate')\n",
-    "p.scatter(df[\"sample_qc.dp_mean\"], df[\"sample_qc.call_rate\"],\n",
-    "            alpha=0.5)\n",
-    "show(p)"
+    "plot.scatter(df[\"sample_qc.dp_mean\"], \n",
+    "            df[\"sample_qc.call_rate\"], \n",
+    "            xlabel='Mean DP', ylabel='Call Rate')"
```

Comment:
I so think we should stop using pandas for any plotting in the tutorial.  You can write this as:

```
(dp_mean, call_rate) = ds.aggregate_cols((ds.sample_qc.dp_mean, ds.sample_qc.call_rate))
p = hl.plot.scatter(dp_mean, call_rate, xlabel=...)
show(p)
```

And eventually, in v2 of this, we will want to write:

```
p = hl.plot.scatter(ds.sample_qc.dp_mean, ds.sample_qc.call_rate, xlabel=...)
show(p)
```

Reply:
@tpoterba

Reply:
this needs to be 
```
(dp_mean, call_rate) = ds.aggregate_cols(
    (hl.agg.collect(ds.sample_qc.dp_mean), 
     hl.agg.collect(ds.sample_qc.call_rate)))
```